src/test/java/com/wikia/webdriver/testcases/mercurytests/LightboxTests.java

```diff
@@ -112,11 +114,13 @@ public void MercuryLightboxTest_003_ZoomByGesture_ZoomByDoubleTap() {
     touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_IN, 3000);
     File afterZooming = new Shooter().capturePage(driver);
```

Comment:
What do you think about converting `File` to `BufferedImage`, since it is a screenshot from viewPort and to use `ImageComparison::areImagesDifferent` method. 
From my point of view, it's more transparent to use comparison of pixels in images than bytes in files.